File: Beecrowd/Beginner/1000 - 1099/1099 - Sum of Consecutive Odd Numbers II/C++/1099_SumofConsecutiveOddNumbersII.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int testCases;
    cin >> testCases;

    while (testCases--) {
        int number1, number2;
        cin >> number1 >> number2;

        int sum = 0;

        if (number1 > number2) {
            for (int j = number2 + 1; j < number1; j++) {
                if (j % 2 != 0) {
                    sum += j;
                }
            }
        } else {
            for (int j = number1 + 1; j < number2; j++) {
                if (j % 2 != 0) {
                    sum += j;
                }
            }
        }

        cout << sum << endl;
    }
    
    return 0;
}

File: Beecrowd/Beginner/1000 - 1099/1099 - Sum of Consecutive Odd Numbers II/Python/1099_SumofConsecutiveOddNumbersII.py
testCases = int(input())

for _ in range(testCases):
    number1, number2 = map(int, input().split())

    sum = 0

    if number1 > number2:
        for j in range(number2 + 1, number1):
            if j % 2 != 0:
                sum += j
    else:
        for j in range(number1 + 1, number2):
            if j % 2 != 0:
                sum += j

    print(sum)
